### ISSUE TITLE
[Feature request] Add option to add LN nodeID

### ISSUE THREAD

**Issue by Darth-Coin** [teambtcmap/btcmap.org#67]
### Description
Option to add / see merchant nodeID (if they run a LN node).

We all know that merchants need the most inbound liquidity.
Those users that want to buy from these merchants, simply can open channels with their nodes. Even if the user have a routing node or a private node, these merchants con promote their nodes too. Everybody wins.
Indeed, not all merchants will want that, but some for sure they will welcome some inbound liquidity.

### Solution suggested
- Add on the btcmap.org page a section where merchants can post their nodeIDs and a link to their shop on the map
or
- add the field for OSM input to add a nodeID. When user pick a place, also can see their node ID

### Additional input
Think about mobile node wallets like: Blixt, OBW, Breez, Phoenix, Electrum. Users of these can easily open up a "spending channel" with these merchant's nodes.

**Comment by secondl1ght** [comment]
This is a really cool idea!

The only problem with this is verifying that the information is accurate - technically anyone could add or change the node information on a merchant since OSM is editable by anybody. It's hard to know if the information is actually provided by the owner or by somebody else and is potentially inaccurate.

**Comment by Darth-Coin** [comment]
Indeed, we need good vouched info. Maybe just create a section on the website, where you verify the info they sent to you?
Not sure if they will be too many, but is good to have a list of real merchant's nodes.

**Comment by dadofsambonzuki** [comment]
Nice idea.

We'd do this as a private tag within the BTCMap backend so that it's not
publicly editable. Just like we do with payment tags.

I'll add it to the roadmap.

**Comment by Darth-Coin** [comment]
Yeah, even better. A more direct p2p thing to connect each others, not widely public.
I asked about this, because writing [this guide](https://darthcoin.substack.com/p/bitcoin-private-banks-over-lightning) and suggesting to use btcmap.org, came to my mind that users do not know to which merchant node could connect their node. So what better option would have than just go to btcmap.org web and choose the right merchant node?

**Comment by secondl1ght** [comment]
Yeah we do have the ability to have tags in our own backend which is how we are handling the current pay buttons but there is still an issue of verification. Somebody could still take a picture outside the business and claim to be the owner, it's a bit tricky... 

Right now we are only adding pay buttons for merchants that we have direct contact with the payment provider to confirm the authenticity, but I'm not sure how well this modal will scale. For now though it's good enough and we could definitely use this method for the node info as well. It's essentially a web of trust modal where somebody reputable has to personally know the merchant to confirm. 

Thanks for including BTC Map in your guide! 

We should probably add the Pay button and this new Node ID info to the Wiki so merchants know it is an option. But @Darth-Coin if you know of any merchants who would like to start using this feature let me know and I can set it up with them - it won't take long to add the tags for a few merchants and display it somewhere in the popup.

**Comment by dadofsambonzuki** [comment]
Yep, merchant verification is our next big headache as we scale.
